akka-cluster-metrics/src/main/scala/akka/cluster/metrics/ClusterMetricsCollector.scala

```diff
@@ -127,7 +127,9 @@ private[metrics] class ClusterMetricsCollector extends Actor with ActorLogging {
   import ClusterMetricsCollector._
   import ClusterEvent._
   import Member.addressOrdering
-  import context.dispatcher
+  import scala.concurrent.ExecutionContext
+
+  private implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
Just used for two tasks scheduled upon actor construction, so fine to keep original